### PR TITLE
modulesync 4.0.0

### LINKED ISSUE #15: Add Puppet version support range ` 5.5.8 < 7.0.0` to metadata.json
this is our current standard range. It should be added so people know what we support. This is fixed with https://github.com/voxpupuli/puppet-xmlfile/pull/14/commits/bacbfdd248686f7e5e6f2b9a59aa162061894d80

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 4.0.0

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
 ---
-inherit_from: '.rubocop_todo.yml'
```
Comment (ekohl) [inline .rubocop.yml:2]: If this is removed, the todo file itself should also be removed.

**Review by ekohl — CHANGES_REQUESTED** [review]
No Puppet requirement found

**Review by ekohl — APPROVED** [review]
:+1: if it passes, but I also made some changes. Wondering if that deserves its own PR for the changelog.

**Comment by bastelfreak** [comment]
created https://github.com/voxpupuli/puppet-xmlfile/issues/15 which mentions the puppet version range

**Pull request merged**